package.json

```diff
@@ -80,7 +80,7 @@
     "vhost": "~3.0.2"
   },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": "^8.9.0 || ^10.13.0 || >=12.0.0"
```

Comment:
What's the reasoning behind these specific versions? 🤔

Reply:
LTS starts at 8.9.0 and 10.13.0. LTS does not exists in 9 and 
11.  https://nodejs.org/en/download/releases/

Reply:
I agree it should not be so specific. I think `>= 8` would be good enough?